dlt/pipeline/pipeline.py

```diff
@@ -294,7 +294,7 @@ def extract(
     @with_runtime_trace
     @with_schemas_sync
     @with_config_section((known_sections.NORMALIZE,))
-    def normalize(self, workers: int = 1, loader_file_format: TLoaderFileFormat = None) -> NormalizeInfo:
+    def normalize(self, workers: int = 1, loader_file_format: TLoaderFileFormat = None, schema_contract_settings: TSchemaContractSettings = None) -> NormalizeInfo:
```

Comment:
OK now I see that this setting does not make sense here OR we apply this to all the schemas in the pipeline. WDYT?

Reply:
moved it to extract